docs/xdr/features/collect/integrations/endpoint/sekoiaio.md

```diff
@@ -339,4 +339,25 @@ agent-latest.exe <command> [<args>]
 We monitor the agent metrics and try to keep its footprint as small as possible. 
 Right now, our agent uses on average less than 3% of CPU and less than 1% of RAM.
 
+## Troubleshoot
+Check the logs
+
+|OS|path|
+|--|--|
+|Windows|C:\Windows\System32\config\systemprofile\AppData\Local\SEKOIA.IO\EndpointAgent\logs|
+|Linux|/var/log/endpoint-agent/agent.log|
+
+!!! note
+    	If you need further assistance, here are the information to provide
+
+1- Sekoia.io agent version (the latest version must be used)
+```
+agent-latest.exe --version
```

Comment:
We must check the version of the installed agent not the one that was downloaded